### DISCUSSION TITLE
Patterned Flow Cell

### DISCUSSION THREAD

**Discussion by josesanchez1815** [nih-cfde/training-and-engagement#448] [RNA - Seq] [answered]
If you take the same library and run some in the patterned flow cell and in the non patterned flow cell, what are the differences you would see in the final results?

**Comment by ACharbonneau** [comment, marked as answer]
You can read a paper about exactly that comparison here: https://sequencing.qcfail.com/articles/illumina-patterned-flow-cells-generate-duplicated-sequences/
